evReady/src/pkix/lib/pkixbuild.cpp

```diff
@@ -221,7 +221,9 @@ PathBuildingStep::Check(Input potentialIssuerDER,
   // responders are allowed to forget about expired certificates, and many OCSP
   // responders return an error when asked for the status of an expired
   // certificate.
-  if (deferredSubjectError != Result::ERROR_EXPIRED_CERTIFICATE) {
+  // As of 2022, Firefox does not check OCSP on intermediate certificates.
+  if (deferredSubjectError != Result::ERROR_EXPIRED_CERTIFICATE &&
+      subject.endEntityOrCA == EndEntityOrCA::MustBeEndEntity) {
```

Comment:
In case this copy of mozilla::pkix is ever updated in the future, this check should probably go in `EVCheckerTrustDomain::CheckRevocation`, but it's not a big deal.